media-proxy/src/mesh/conn_rdma.cc

```diff
@@ -0,0 +1,388 @@
+#include <queue>
+#include "conn_rdma.h"
+
+namespace mesh {
+
+namespace connection {
+
+Rdma::Rdma() : ep_ctx(nullptr), init(false), trx_sz(0), mDevHandle(nullptr)
+{
+    std::memset(&ep_cfg, 0, sizeof(ep_cfg));
+}
+
+Rdma::~Rdma() { shutdown_rdma(context::Background()); }
+
+std::string Rdma::kind_to_string(Kind kind)
+{
+    switch (kind) {
+    case Kind::undefined:
+        return "undefined";
+    case Kind::transmitter:
+        return "transmitter";
+    case Kind::receiver:
+        return "receiver";
+    default:
+        return "unknown";
+    }
+}
```

Comment:
Replace with `kind2str()` from conn.cc.

Reply:
fixed in https://github.com/OpenVisualCloud/Media-Communications-Mesh/pull/286/commits/1e7f66891946f5a904408fb568ca119b7e04eaa0